backend/test/integration/test_general.py

```diff
@@ -32,6 +32,8 @@ def test_health_for_locale(self, with_client):
     def test_access_without_authentication_token(self, with_client):
         # response = client.get("/login")
         # assert response.status_code == 401
+        response = with_client.get('/me')
```

Comment:
We could probably adjust this to be a list of routes that we then iterate over and check for 401.